app/main.py

```diff
@@ -1,3 +1,50 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union[int, "Distance"]) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
+            return Distance(self.km + other)
+
+    def __iadd__(self, other: Union[int, "Distance"]) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: Union[int, "Distance"]) -> "Distance":
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: Union[int, "Distance"]) -> "Distance":
+        return Distance(round(self.km / other, 2))
+
+    def __eq__(self, other: Union[int, "Distance"]) -> bool:
+        if isinstance(other, Distance):
+            return self.km == other.km
+        return self.km == other
+
+    def __lt__(self, other: Union[int, "Distance"]) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        return self.km < other
+
+    def __ge__(self, other: Union[int, "Distance"]) -> bool:
+        return not self.__lt__(other)
+
+    def __gt__(self, other: Union[int, "Distance"]) -> bool:
+        return not self.__lt__(other) and not self.__eq__(other)
+
+    def __le__(self, other: Union[int, "Distance"]) -> bool:
+        return self.__lt__(other) or self.__eq__(other)
```

Comment:
You can use same operands as previously " <>= " to make it same type and more readable